elementary/utils/log.py

```diff
@@ -46,7 +61,10 @@ def get_logger(logger_name):
     logger = logging.getLogger(logger_name)
     logger.setLevel(logging.DEBUG)
     logger.addHandler(get_console_handler())
-    logger.addHandler(get_file_handler())
     # with this pattern, it's rarely necessary to propagate the error up to parent
-    logger.propagate = False
     return logger
+
+
+def set_root_logger_file_handler(logger_name):
```

Comment:
I think you should add both handlers only to the root logger, otherwise you'll get duplicate console outputs. 
So maybe we can have `set_root_logger_handlers`?

Reply:
I actually don't see it log twice to the console so it's interesting
But anyway I'll move the console handler to the root as well, I agree it's more sorted that way